packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton.tsx

```diff
@@ -62,24 +73,81 @@ export const RainbowKitCustomConnectButton = () => {
+                      <li>
+                        {addressCopied ? (
+                          <div>
+                            <CheckCircleIcon
+                              className=" text-xl font-normal h-6 w-4 cursor-pointer"
+                              aria-hidden="true"
+                            />
+                            <span className=" whitespace-nowrap">Copy address</span>
+                          </div>
+                        ) : (
+                          <CopyToClipboard
+                            text={account.address}
+                            onCopy={() => {
+                              setAddressCopied(true);
+                              setTimeout(() => {
+                                setAddressCopied(false);
+                              }, 800);
+                            }}
+                          >
+                            <div>
+                              <DocumentDuplicateIcon
+                                className=" text-xl font-normal h-6 w-4 cursor-pointer"
+                                aria-hidden="true"
+                              />
+                              <span className=" whitespace-nowrap">Copy address</span>
+                            </div>
+                          </CopyToClipboard>
+                        )}
+                      </li>
+                      <li>
+                        <label htmlFor="qrcode-modal">
+                          <QrCodeIcon className="h-6 w-4 ml-2 sm:ml-0" />
+                          <span className="whitespace-nowrap">View QR Code</span>
+                        </label>
+                      </li>
+                      <li>
+                        <button className="menu-item text-error" type="button" onClick={() => disconnect()}>
+                          <ArrowLeftOnRectangleIcon className="h-6 w-4 ml-2 sm:ml-0" /> <span>Disconnect</span>
+                        </button>
+                      </li>
+                    </ul>
+                  </div>
+                  <div>
+                    <input type="checkbox" id="qrcode-modal" className="modal-toggle" />
+                    <label htmlFor="qrcode-modal" className="modal cursor-pointer">
+                      <label className="modal-box relative">
+                        {/* dummy input to capture event onclick on modal box */}
+                        <input className="h-0 w-0 absolute top-0 left-0" />
+                        <label
+                          htmlFor="qrcode-modal"
+                          className="btn btn-ghost btn-sm btn-circle absolute right-3 top-3"
+                        >
+                          ✕
+                        </label>
+                        <div className="space-y-3 py-6">
+                          <div className="flex space-x-4 flex-col items-center gap-6">
+                            <QRCodeSVG value={account.address} size={256} />
```

Comment:
NITPICK : 

Maybe give `className="rounded-xl"` to `QRCodeSVG` ? So that it matches Modal & our kind of general rounded theme 

![Screenshot 2023-07-25 at 9 43 56 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/57ac7d9e-d6aa-46f2-92aa-e3326ddfbe5c)

PS : I think we can also remove the parent's div `space-y-3`

Reply:
Looks a lot better, thanks!